tests/Unit/Domain/Test_Domain.cpp

```diff
@@ -44,7 +45,30 @@ namespace helpers = ::TestHelpers::domain::BoundaryConditions;
 
 void test_1d_domains() {
   using Translation = domain::CoordinateMaps::TimeDependent::Translation<1>;
+  using TranslationGridDistorted =
+      domain::CoordinateMap<Frame::Grid, Frame::Distorted, Translation>;
+  using TranslationDistortedInertial =
+      domain::CoordinateMap<Frame::Distorted, Frame::Inertial, Translation>;
   {
+    using logical_to_grid_coordinate_map =
```

Comment:
when rebasing can you change these type aliases to `CamelCase`? They're being used as classes instead of metavariables :)